chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <deploy_image>
+    <image_installation config:type="boolean">false</image_installation>
+  </deploy_image>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <final_halt config:type="boolean">false</final_halt>
+      <final_reboot config:type="boolean">false</final_reboot>
+      <halt config:type="boolean">false</halt>
+      <second_stage config:type="boolean">true</second_stage>
+    </mode>
+    <mouse>
+      <id>none</id>
+    </mouse>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage/>
+  </general>
+  <add-on>
+    <add_on_products config:type="list">
+    <% @repos.keys.sort.each do |name| %>
+      <listentry>
+        <name><%= name %></name>
+        <product><%= name %></product>
+        <product_dir>/</product_dir>
+        <media_url><%= @repos[name][:url] %></media_url>
+        <!-- See bnc#787157 for discussion on ask_on_error -->
+        <ask_on_error config:type="boolean"><%= @repos[name][:ask_on_error] || 'false' %></ask_on_error>
+      </listentry>
+    <% end %>
+    </add_on_products>
+  </add-on>
+  <timezone>
```

Comment:
Yeah, I was unsure about this one. But it's an allowed autoyast section for the upgrade and doesn't really hurt. I can remove it though if you really insist :)